.travis.yml

```diff
@@ -36,9 +36,14 @@ branches:
 sudo: false
 
 addons:
-  # make sure simplehttp simple verification works (custom /etc/hosts)
+  # Custom /etc/hosts required for SimpleHTTP simple verification,
```

Comment:
`SimpleHTTP` is not a thing anymore, it has been renamed to `http-01`, please adjust comment